src/com/shmuelzon/HomeAssistantFloorPlan/Controller.java

```diff
@@ -77,8 +80,32 @@ public Controller(Home home) {
         lightsNames = new ArrayList<String>(lights.keySet());
         lightsPower = getLightsPower(lights);
         homeAssistantEntities = getHomeAssistantEntities();
+        this.quality = Quality.HIGH;
+    }
+
+    public Home getHome() {
```

Comment:
I'd rather not expose `Home` directly to the UI. If you really want to indicate the project is empty, create a function specifically for that